mlfinlab/portfolio_optimization/risk_estimators.py

```diff
@@ -0,0 +1,259 @@
+# pylint: disable=missing-module-docstring
+import numpy as np
+import pandas as pd
+from sklearn.neighbors import KernelDensity
+from scipy.optimize import minimize
+
+
+class RiskEstimators:
+    """
+    This class implements the functions related to the de-noising of the covariance matrix. It is reproduced with
+    modification from the following paper: `Marcos Lopez de Prado “A Robust Estimator of the Efficient Frontier”,
+    (2019). <https://papers.ssrn.com/abstract_id=3469961>`_.
+    """
+
+    def __init__(self):
+        """
+        Initialize
+        """
```

Comment:
Just add a return statement to avoid any blank lines

Reply:
Done